Templating/TemplateGuesser.php

```diff
@@ -30,22 +30,33 @@ class TemplateGuesser
     protected $kernel;
 
     /**
+     * @var string[]
+     */
+    private $controllerPatterns;
+
+    /**
      * Constructor.
      *
-     * @param KernelInterface $kernel A KernelInterface instance
+     * @param KernelInterface $kernel             A KernelInterface instance
+     * @param string[]        $controllerPatterns Regexps extracting the controller name from its FQN.
      */
-    public function __construct(KernelInterface $kernel)
+    public function __construct(KernelInterface $kernel, array $controllerPatterns = null)
     {
+        if (null === $controllerPatterns) {
+            $controllerPatterns = array('/Controller\\\(.+)Controller$/');
```

Comment:
I think we need to always add this to the list of patterns. If not, as soon as you define the configuration setting, bundles won't work anymore, which would be really hard to understand for the end user. Any downside to always add this?